src/grt/src/fastroute/src/utility.cpp

```diff
@@ -319,6 +363,31 @@ void FastRouteCore::fillVIA()
   }
 }
 
+/*returns the start and end of the stack necessary to reach a node*/
+void FastRouteCore::viaStack(int netID,
```

Comment:
The function is not creating the via stack. Perhaps a more descriptive name would be better, like `findViaStackRange` or something similar.